linux_docker_resources/Dockerfile

```diff
@@ -12,7 +12,8 @@ ARG COMPILE_WITH_CLANG=false
 ENV DEBIAN_FRONTEND noninteractive
 
 RUN apt-get update && apt-get install --no-install-recommends -y locales
-RUN locale-gen en_US.UTF-8
+RUN if test ${UBUNTU_DISTRO} != buster; then locale-gen en_US.UTF-8; fi
+RUN if test ${UBUNTU_DISTRO} = buster; then sed -i -e 's/# en_US.UTF-8 UTF-8/en_US.UTF-8 UTF-8/' /etc/locale.gen && dpkg-reconfigure --frontend=noninteractive locales; fi
```

Comment:
It's not great for readability, but I think these conditionals should be combined into one RUN invocation unless there's a reason they couldn't be.